starter-code/scripts/models/article.js

```diff
@@ -43,10 +43,18 @@ Article.loadAll = function(inputData) {
  source, process it, then hand off control to the View: */
 Article.fetchAll = function() {
   if (localStorage.blogArticles) {
+    Article.loadAll(JSON.parse(localStorage.blogArticles));
+    articleView.renderIndexPage();
+
     /* When our data is already in localStorage:
     1. We can process and load it,
     2. Then we can render the index page.  */
   } else {
+    $.getJSON('data/blogArticles.json', function(data) {
+      localStorage.blogArticles = JSON.stringify(data);
+      Article.loadAll(JSON.parse(localStorage.blogArticles));
```

Comment:
👍  Anything equivalent we could have passed into `loadAll()` to achieve the same result?